include/mbgl/gl/gl_defines.hpp

```diff
@@ -0,0 +1,197 @@
+#pragma once
```

Comment:
Why are these files public?

Reply:
It is needed by `include/mbgl/gl/gl_function_pointers.hpp` to define the function pointers to be implemented by the platform.

Reply:
Ok, but can we try to make them private? Especially in light of upcoming changes wrt graphics backend, I think they should remain private.

Reply:
I could move `using GLbitfield = unsigned int; ...` to `gl_function_pointers.hpp` and make the defines private.